goverance.md

```diff
@@ -0,0 +1,67 @@
+
+## Creating, discussing and accepting proposals
+
+Each proposal is unique and might deviate slightly from the process below. For example, a smaller project may each step. For example, a small addition may not require establishing completion criteria. In general, we encourage this process below to be followed to ensure that contributions are in line with the goals of the Working Group.
+### Proposal process
+
+1. **Raise an Issue.** [Create an issue](https://github.com/cn-security/safe/issues/new) in the SAFE repo that outlines the problem to be solved. If possible, include:
+  * The customer impact of the problem
+  * The scope of the work required
+
+1. **Ask the group for collaboration.** Rather than immediately beginning work on a solution, at the next meeting, bring the issue up for discussion, explain it, and ask if others are interested in collaborating. This ensures that solutions are created with multiple perspectives. The outcome of this conversation will be:
+  * The group agrees that this is a problem to solve
+  * There are criteria for completion or failure, and those will be added to the issue.
+  * At least one person is tasked with creating a solution
+  * Those interested in working on the solution either begin work or set up time or expectations with others to begin work.
+  * A TL is agrees to actively sponsor the work.
+
+1. **Track progress.** As long as work is ongoing, progress should be tracked in the issue.   
+  * Collaborators should update the body of the Issue or PR to include a checklist of work that shows what has been done and what work remains.
+  * Additionally, when possible, someone working on the project should attend the weekly meetings and give an update.
+
+1. **Discuss the work at a meeting.** If an objection to a PR is made either in a comment of the PR or in call discussing the meeting, both the person making the objection and the person making the person making the proposal will have time to present their case at the next meeting. If there are not unaddressed concerns, and the Pull Request has been stable for 48 hours, a Chair will add it to the agenda for an upcoming meeting. Ideally, members who contributed to the project will attend that meeting to present their work or answer questions.
+
+1. **Vote, if required.** If there’s not (lazy) consensus among the group, a formal vote is required. Only one member from each company should vote. Members will vote by leaving a comment in the Pull Request to indicate their vote for or against. Members will have a week to vote.
+
+1. **Accept or close the proposal.** Depending on the outcome of a discussion, a Chair will merge or close the work created. When a proposal is closed, the group should create a plan to address the underlying user problem that the proposal addressed.
+
+1. **Support the project going forward.** Some projects require ongoing support. When the work is completed
```

Comment:
Is there more that is going to be said here?

Reply:
The current version reads: 
> **Support the project going forward.** Some projects require ongoing support. When the work is completed, the Pull Request should specify if it's expected to require on-going support and if so, who volunteers to maintain it.

The step `Ask the group for collaboration`, now includes specifying if the project is expected to be on-going or time-bounded.